rendezvous/internal/msgs/value.go

```diff
@@ -0,0 +1,5 @@
+package msgs
+
+type RendezvousValue interface {
+	RendezvousValue() string
```

Comment:
Please use a better name for this function, maybe something that is related to "type", "kind" or "tag" since the function basically represents a type discriminant.

Reply:
Indeed. I have already fixed that locally when trying to remove even more uses of reflect. Will try to get that code working and mark this as draft in the meantime.

Reply:
I think it should be fixed now. Feel free to suggest even better interface names though :)